test/datalog_tests/redist_opt.dl

```diff
@@ -0,0 +1,142 @@
+import tinyset as TS
```

Comment:
is it ok to make this public?

Reply:
it's just reachability in a graph, so should be ok.